mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.EraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
+      }
+      res.json({ success: "Order has been deleted" });
+    })
+    .catch(error => {
```

Comment:
The error variable should be logged or handled more explicitly, instead of just passing it to the errorHandler function. Consider adding a console.error or a more robust error logging mechanism.